android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -169,6 +230,57 @@
     private List<NTPWidgetItem> widgetList = new ArrayList<NTPWidgetItem>();
     public static final int NTP_WIDGET_STACK_CODE = 3333;
 
+    // Brave news
+    private BraveNewsAdapterFeedCard adapterFeedCard;
+    private FrameLayout optinButton;
+    private TextView optinText;
+    private LinearLayout optinLayout;
+    private TextView optinLearnMore;
+    private ImageView optinClose;
+    private CopyOnWriteArrayList<FeedItemsCard> newsItemsFeedCard =
+            new CopyOnWriteArrayList<FeedItemsCard>();
+    private LinearLayout container;
+    private RecyclerView recyclerView;
+    private TextView loading;
+    private View loadingView;
+    private View feedSpinner;
+    private ScrollView parentScrollView;
+    private ViewGroup imageCreditLayout;
+    private ViewGroup settingsBar;
+    private ViewGroup newContentButton;
+    private boolean isScrolled;
+    private NTPImage ntpImageGlobal;
+    private boolean settingsBarIsClickable;
+    private BraveNewsController mBraveNewsController;
+    private BraveNewsUtils braveNewsUtils;
```

Comment:
with the https://github.com/brave/brave-core/pull/11121/files#r754958588
you will be able to remove it

Reply:
I see this is also fixed, thanks